README_EN.md

```diff
@@ -0,0 +1,233 @@
+end
+```
+
+Then, run the following command:
+
+```bash
+$ pod install
+```
+
+
+## Usage
+
+You can take a look at the example in the RexxarDemo folder.
+
+We use github raw file to serve the routes map api for this demo. You definitely need implement a real server in your real product serving the routes map api and html, css, javascript resource files. 
+
+You can use any server framework. Rexxar hasn't any requirement about the server framework. Rexxar offers the configuration interface of the routes map api address in `RXRConfig`. You can find the way to configure the address in next section.
+
+### Configure routes map api address
+
+```Swift
+  RXRConfig.setRoutesMapURL(NSURL(string:"https://raw.githubusercontent.com/douban/rexxar-web/master/example/dist/routes.json)!)
+```
+
+Rexxar use url to identify the page in mobile application. With a valid url, we can create a RXRViewController. There is the map from url to html resources in the routes map api file. In the Demo, you can see a routes map api file like this:
+
+```json
+{
+    "items": [{
+        "remote_file": "https://raw.githubusercontent.com/douban/rexxar-web/master/example/dist/rexxar/demo-252452ae58.html",
+        "deploy_time": "Sun, 09 Oct 2016 07:43:47 GMT",
+        "uri": "douban://douban.com/rexxar_demo[/]?.*"
+    }],
+    "partial_items": [{
+        "remote_file": "https://raw.githubusercontent.com/douban/rexxar-web/master/example/dist/rexxar/demo-252452ae58.html",
+        "deploy_time": "Sun, 09 Oct 2016 07:43:47 GMT",
+        "uri": "douban://partial.douban.com/rexxar_demo/_.*"
+    }],
+    "deploy_time": "Sun, 09 Oct 2016 07:43:47 GMT",
+}
+```
+
+### Configure the resource path
+
+```Swift
+  RXRConfig.setRoutesResourcePath("rexxar")
+```
+
+Using Rexxar to develop mobile application, we usually put a copy of resource files in the application's release package. This copy of resource files in app's package will reduce user's download of resource files and optimise the user's experience of application.
+
+If we configure the resource path in app's package of Rexxar, the type of folder including the resource files must be `folder reference` which is blue folder icon in Xcode, rather than yellow folder icon of `group` type.
+
+### Configure the cache path
+
+```Swift
+  RXRConfig.setRoutesCachePath("com.douban.RexxarDemo.rexxar")
+```
+
+The cache can also reduce user's download of resource files and optimise the user's experience of application.
+
+The cache will be created after Rexxar Container gets the update of routes map file. 
+
+The routes map file decides the update of resouces files. Application visits the routes map api periodically. If it gets the update of routes map, Rexxar Container will download the update of resource files and save the copy in the cache path. That's the way Rexxar supports `hot deploy`. You don't need release your application via App Store. In stead of that, Rexxar update the pages after the update of routes map file.
```

Comment:
Cache is also used to improve performance over routing. Rexxar will check the routes map periodically and save the latest version in the cache. By deploying different path with the routes, it's possible to do a `hot deployment`, with which you can update the page just by replacing the resource file, and no need to go through the tedious process of asking a new release via App Store.